script/workLoggingSystem.js

```diff
@@ -140,11 +140,60 @@ $("#submitNewTask").click(function() {
     console.log(estimatedTime);
     console.log(status);
     console.log(category);
+    if(checkValidation(priority,difficulty,status,category,estimatedTime))
+        {
+            console.log("working");
+            createNewTask(title,des,priority,difficulty,creator,assignee,estimatedTime,status,category);
+        }
+    else
+        {
+            console.log("wrong values");
+            $("#errorMessage").show();
+          
```

Comment:
Incorrect else statement, please learn basic coding

Reply:
the closing bracket is missing due to the conflict merge